### DISCUSSION TITLE
Converting WPF to UNO: Oxyplot and HelixToolKit

### DISCUSSION THREAD

**Discussion by bharathkondam** [unoplatform/uno#18122] [General]
Exploring the use of UNO, for converting WPF application, what options do I have for Oxyplot and HelixToolKit?

**Comment by jeromelaban** [comment]
Thanks for the question. Oxyplot is supported by Uno, HelixToolKit has no direct equivalent yet, though we're working on 3D integration using OpenGL (https://github.com/unoplatform/uno/issues/9405).
